src/plugins/timer/actions/PauseTimerAction.js

```diff
@@ -39,7 +39,7 @@ export default class PauseTimerAction {
 
     const newConfiguration = { ...domainObject.configuration };
     newConfiguration.timerState = 'paused';
-    newConfiguration.pausedTime = new Date();
+    newConfiguration.pausedTime = new Date(this.openmct.time.now());
```

Comment:
The uses of `Date` here sets off my spidey senses.  The Time API supports time systems that are not directly parseable by JavaScript Dates (eg. SCET, LMST, etc.). That said, writing clocks and timers that support those time systems is a whole other deal...